cinema/tests/test_movie_session_api.py

```diff
@@ -117,9 +104,7 @@ def test_get_movie_session(self):
         response = self.client.get("/api/cinema/movie_sessions/1/")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.data["movie"]["title"], "Titanic")
```

Comment:
Code Efficiency: It seems like you're splitting the date in your tests. Remember, it's already in the format needed. You should be able to just filter by the date directly. For example:

```python
movie_sessions = self.client.get("/api/cinema/movie_sessions/?date=2022-09-02")
```

This makes your code more efficient and easier to read.